Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG LUET_VERSION=0.35.2
-ARG GO_VERSION=1.22-alpine
+ARG GO_VERSION=1.22.5-alpine
```

Comment:
why does it need the .5? shouldn't it pull the latest with 1.22?

Reply:
it should but somehow it says that its not 1.22.5 but 1.22.4 so it may be that the tag was not updated upstream?